src/toniarts/openkeeper/game/controller/IInteractiveControl.java

```diff
@@ -64,26 +63,27 @@ public enum DroppableStatus {
      */
     public IInteractiveControl pickUp(short playerId);
 
-    /**
-     * Get the result of dropping the control on a tile. A result what would
-     * happen if...
-     *
-     * @param tile the tile to be tested against
-     * @param playerId the player who wants to drop
-     * @return the result if the control were to be dropped here
-     */
-    public DroppableStatus getDroppableStatus(TileData tile, short playerId);
+    /**
+     * Get the result of dropping the control on a tile. A result what would
+     * happen if...
+     *
+     * @param tile the tile to be tested against
+     * @param playerId the player who wants to drop
+     * @return the result if the control were to be dropped here
+     */
+    // TODO: Replace TileData with appropriate alternative from new system
+    // public DroppableStatus getDroppableStatus(TileData tile, short playerId);
 
-    /**
-     * Drop the control on the tile
-     *
-     * @param tile tile to drop to
-     * @param coordinates real world coordinates inside
-     * @param control if there is already an
-     * {@link toniarts.openkeeper.world.control.IInteractiveControl} at the
-     * position
-     */
-    public void drop(TileData tile, Vector2f coordinates, IInteractiveControl control);
+    /**
+     * Drop the control on the tile
+     *
+     * @param tile tile to drop to
+     * @param coordinates real world coordinates inside
+     * @param control if there is already an
+     * interactive control at the position
+     */
+    // TODO: Replace TileData with appropriate alternative from new system
```

Comment:
Yeah you just removed everything but the tricky part is figuring out how to replace things.